src/test-run/index.js

```diff
@@ -228,5 +240,5 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
         }
     }
 
-    return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
+    return new Promise((resolve, reject) => this.pendingRequest = { statusId: driverStatus.id, resolve, reject });
```

Comment:
Thanks for this. It seems I've found an uncovered scenario after reviewing this and maybe have found a way to test it